src/components/Verification/EmailVerified.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+import * as React from 'react';
+
+// components
+import Button from 'components/Button';
+import { MediumText } from 'components/Typography';
+
+type Props = {
+  onPressBack: () => void,
+};
+
+const EmailVerified = (props: Props) => {
+  const { onPressBack } = props;
+
+  return (
+    <React.Fragment>
+      <MediumText>
+        You have confirmed your email!
```

Comment:
Please add space between button and text. Also the text could be a bit bigger (on all verification screens).

<img width="410" alt="Screenshot 2020-02-19 at 11 15 28" src="https://user-images.githubusercontent.com/6280457/74825421-327c3280-530a-11ea-9000-2f25cf4b9e9b.png">

Reply:
Updated. Created a `Label` subcomponent so we have all of them look the same and we can easily update it. Not sure if there's already a component that we could have used for this (The `Typography/Label` doesn't look like a fit, let me know what you think.

Reply:
It's a shame that this branch is not updated with develop, because on develop we have those new Typography components and you'd just do something like this:
`<MediumText big>...</MediumText>`. And creating a new separate component for some text on one screen is a bit overkill. And the name is too generic as well. 
Here is what'd do to: looking at the designs, such "descriptions" are usually regular font with size 16sp (see for example the recovery flow, there are a lot of similar screens with description and a button). And in the design system such font is marked as just "Text". So I'd create a new component in the Typography file called "Text" or "RegularText" (so it follows the design system) and use that. How does it sound?